reana_server/status.py

```diff
@@ -599,11 +600,11 @@ def __init__(self):
         self.session = self.get_session_health()
 
     @staticmethod
-    def get_health_status(percentage):
+    def get_health_status(percentage, total):
         """Calculate quota health status."""
         health = ClusterHealthStatus.healthy
         if percentage >= 50:
-            if percentage >= 75:
+            if percentage >= 75 and total > REANA_MAX_CONCURRENT_BATCH_WORKFLOWS:
```

Comment:
`get_health_status` is a very generic method that ignores if we're calculating the Workflow or the Job health.

If I'm not wrong, [here](https://github.com/reanahub/reana-server/pull/419/files#diff-c23f1238a97f03b453a1be48238f557dcc0ce18c1ee6ba8992297d149ff14f19R649) we're passing the total number of jobs so we'd be comparing jobs with workflows, which might cause unexpected results.

Reply:
You are right, it's not nice to compare jobs with workflows. Idea was to add some buffer for the `critical` status. If by default we have `REANA_MAX_CONCURRENT_BATCH_WORKFLOWS` set to 30, only if there are more jobs launched and most of them are pending (>=75%), only in this case it will be reported as `critical`. This is to prevent 2 pending jobs out of 3 to be `critical`.. I will try to come up with something next week